keyboards/viendi8l/keymaps/default/keymap.c

```diff
@@ -0,0 +1,276 @@
+/*
```

Comment:
Default keymap should really be as simple as possible.

Reply:
What fauxpark said.  The reasoning for this is that when somebody clones the default keymap, if they don't know what is going on because they're not familiar with C, it can be overwhelming.  

Having a "vendor" keymap with the more complicated stuff would be fine.

Reply:
Comitted in 6c7cc960f5